sources/sections-en/05-unit-symbols.adoc

```diff
@@ -95,8 +95,8 @@ This rule means that the symbol stem:["unitsml(degC)"] for the degree Celsius(((
 ====
 stem:[t = 30.2 "unitsml(degC)"] +
 but not +
-stem:[t = 30.2 "unitsml(degC)"] +
-nor stem:[t = 30.2 "unitsml(degC)"]
+stem:[t = 30.2"°C"] +
+nor stem:[t = 30.2"° C"]
```

Comment:
Leave them as UnitsML. This is a UnitsML rendering issue.

Reply:
@ronaldtse, I disagree on this part.
The point of this text is to shown wrong ways of expressing the `°C` unit.
So, we need to reproduce them exactly how it's depicted in original version.

Original version:
![issue2](https://user-images.githubusercontent.com/33627611/122432099-0820a080-cf63-11eb-8a4b-f27fcf97aed5.PNG)

From BIPM:
![issue2-1](https://user-images.githubusercontent.com/33627611/122431952-e3c4c400-cf62-11eb-87b2-919bca10b214.PNG)

Reply:
Ah, I got confused. Thanks!